roswell/run-parachute.ros

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+#|-*- mode:lisp -*-|#
+#|
+exec ros -Q -- $0 "$@"
```

Comment:
Not everyone uses roswell. Testing a few implementations if roswell is not available would be good.

Reply:
Hmm.  Because it's set up using the conventions for .ros scripts, I'm having issues figuring out how to work around the combination of shebang and main function.  Looking at sbcl's command line options specifically, the ability to ignore the shebang line and the ability to run additional code after the script.

The best solution I'm coming up with for supporting non-Roswell systems is to bundle a second copy of the script that exists as a plain lisp source file.  The code duplication is necessary for this approach because Roswell makes copies of script files it knows about, so referencing "local" files gets problematic.

There's also a slight inconvenience of needing to find the path when not using Roswell, since Roswell places all of the ros scripts of installed through its quicklisp in a specific directory.  Although, it could be downloaded with curl or a similar tool right to a convenient directory by the user.

Reply:
You could feed the remainder of the file (cutting off the first line) through stdin on quite a few of them I think.